utils/types/deployment/deployment.go

```diff
@@ -3,25 +3,30 @@ package deployment
 import (
 	"fmt"
 
+	"github.com/rudderlabs/rudder-server/utils/logger"
+
 	"github.com/rudderlabs/rudder-server/config"
 )
 
 type Type string // skipcq: RVV-B0009
 
 const (
-	DedicatedType   Type = "DEDICATED"
-	MultiTenantType Type = "MULTITENANT"
+	DedicatedType   Type   = "DEDICATED"
+	MultiTenantType Type   = "MULTITENANT"
+	HostedNamespace string = "free-us-1" // Having it here to support legacy cp-router hosted
```

Comment:
Is this used outside this package? If not please unexport it since we can export it later when needed.